README.md

```diff
@@ -191,3 +192,21 @@ be organised in an appropriate sub-folder there.
   device you need BYOD & 2FA set up.
 * A publish user with your email address with access to the organisation(s) you
   want to modify.
+
+### DfE Sign-in
+
+To use it locally (and in qa) you will need a DfE Sign-in account.
+Clicking on `Create an account` asks you to 
+"Contact the approver of your organisation". We're within DfE itself, so
+that's not terribly helpful.
+
+##### How to get approved and get access to test data:
+1. Get a DfE Sign-in admin to invite you (give them this link: 
+https://signin-test-sup-as.azurewebsites.net/users)
+2. Sign up from the email sent from DfE Sign-in
+3. Grant access to providers (here on production): 
+`bundle exec bin/mcb users -E production grant {email} {course_code}`
+
+Note that all users are anonymized and synced to `qa` daily,
```

Comment:
reword it the other way please, all other environments uses anonymized not just `qa` or `localhost`

Reply:
How about "Users not matching `%@digital.education.gov.uk` and `%@education.gov.uk` will be anonymized."?